redigomock.go

```diff
@@ -263,6 +267,27 @@ func (c *Conn) Receive() (reply interface{}, err error) {
 func (c Conn) Stats(cmd *Cmd) int {
 	c.statsMut.RLock()
 	defer c.statsMut.RUnlock()
-	
+
 	return c.stats[cmd.hash()]
 }
+
+// ExpectationsWereMet can guarantee that all commands that was set on unit tests
+// called or call of unregistered command can be caught here too
+func (c Conn) ExpectationsWereMet() error {
+	errMsg := ""
+	for _, err := range c.Errors {
+		errMsg = fmt.Sprintf("%s%s\n", errMsg, err.Error())
+	}
+
+	for _, cmd := range c.commands {
+		if !cmd.Called {
+			errMsg = fmt.Sprintf("%sCommand %s with arguments %#v expected but never called.\n", errMsg, cmd.Name, cmd.Args)
```

Comment:
I only return one error but with multiline messages. notice the \n at the end of the format